### ISSUE TITLE
Running libimobiledevice from an iPad?

### ISSUE THREAD

**Issue by ScottPierce** [libimobiledevice/libimobiledevice#1053]
Is it possible to use this library from an iOS device like an iPad? Has anyone gotten this working?

**Comment by Schlaubischlump** [comment]
Yes its possible to compile the library for iOS. I haven't tested all features on iOS. You can find a script on my repo which sometimes works and sometimes doesn't that helps you to crosscompile the library for iOS.
